orchestrator/workflows/tasks/validate_products.py

```diff
@@ -109,7 +109,7 @@ def check_that_products_have_create_modify_and_terminate_workflows() -> State:
         workflows = {
             c.target
             for c in product.workflows
-            if c.target in ["CREATE", "TERMINATE", "MODIFY", "SYSTEM"] and c.name != "modify_note"
+            if c.target in ["CREATE", "TERMINATE", "MODIFY", "SYSTEM", "VALIDATE"] and c.name != "modify_note"
         }
         if len(workflows) < 4:
```

Comment:
I would expect that this (terrible) magic number would also need an update

Reply:
Good point, seems that this refers to the length of the list so we could create a variable for that list.

Also `SYSTEM` should probably be removed because it's no longer the target for product validation workflows